src/main/java/seedu/address/logic/parser/MarkTodoCommandParser.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MARK_TASK_INDEX;
+
+import java.util.stream.Stream;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.MarkTodoCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new MarkTodoCommand object
+ */
+public class MarkTodoCommandParser implements Parser<MarkTodoCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the MarkTodoCommand
+     * and returns a MarkTodoCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
```

Comment:
blank line

Reply:
Thank you. I will merge the changes and request review again :)